test/proposals/Addresses.sol

```diff
@@ -266,12 +266,12 @@ contract Addresses is Test, ChainIds {
         _addAddress(
             "MWETH_IMPLEMENTATION",
             84531,
-            0x7c8599bE1f71b8eAF9bAD1B1e9D5F8676C1C1958
+            0x30d3740d8d15004E3D14be74E97Fc4BA189C6400
```

Comment:
this contract isn't verified on etherscan

Reply:
Fixed.